### DISCUSSION TITLE
How many writing systems are there in Japanese?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#699] [Galaxy Trivia] [answered]
**Bonus:** Write the pronoun *"I, me"* in all systems.

**Comment by peterbaccaro** [comment, marked as answer]
There are **three** writing systems in Japanese: Hiragana, Katakana, and Kanji.

Hiragana is a phonetic alphabet that is used to write native Japanese words, grammatical particles, and inflections. It consists of 46 characters and is the first writing system that Japanese children learn.

Katakana is also a phonetic alphabet, but it is used to write loanwords from foreign languages, as well as for emphasis and onomatopoeia. Like Hiragana, Katakana has 46 characters.

Kanji, on the other hand, are Chinese characters that were introduced to Japan over a thousand years ago. Kanji are used to write nouns, verbs, and adjectives, and they represent both meaning and pronunciation. There are thousands of Kanji characters, but the Japanese Ministry of Education has designated 2,136 characters as the "joyo kanji" or the "regular-use kanji," which are taught in schools.

Here are the ways to write the pronoun "I, me" in all writing systems:

In Hiragana: あたし (atashi), わたし (watashi)

In Katakana: アイ (ai), ミー (mii)

In Kanji: 私 (わたし, watashi), 僕 (ぼく, boku), 俺 (おれ, ore)

**Reply by ParanoidUser** [reply]
Brilliant! 💎
